functions/package.json

```diff
@@ -12,33 +12,35 @@
     "test": "echo \"No tests yet, feel free to add\" && exit 0"
   },
   "engines": {
-    "node": "12"
+    "node": ">=16.x"
```

Comment:
This needs to be 16.x or 18.x without the greater-than-or-equal, whichever version is used to actually run the functions.

Also feel free to use Volta. It is a great tool that works much more smoothly than `n` and `nvm`, and it can be configured in the package.json